### PR TITLE
add url support

### PR DESCRIPTION
**Pull request by hhsnopek**
requires - https://github.com/carrot/rooftop-node/pull/4

### CODE REVIEW

**Comment by hhsnopek** [comment]
all set @jescalan in both PRs 👍

File: README.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ module.exports = {
 }
 ```
 
+Optionally you may pass `url: 'http://xxx.domain.com'` into the rooftop object as well. This will override the base url `https://xxx.rooftopcms.io` with the passed url.
+
```
Comment (jescalan) [inline README.md]: Since we removed name now I think this has to change

File: lib/index.js [hunk 2]
```diff
@@ -11,7 +11,11 @@ class Rooftop {
   constructor (opts) {
     const validatedOptions = validate(opts)
     Object.assign(this, validatedOptions)
-    this.client = Client.new({ name: this.name, apiToken: this.apiToken })
+    this.client = Client.new({
+      name: this.name,
```
Comment (jescalan) [inline lib/index.js]: ⛔

File: README.md (hunk 1 above)
Comment (hhsnopek) [inline README.md]: I was just about to push up the changes here 😛

**Comment by jescalan** [comment]
👍

**Comment by hhsnopek** [comment]
rebased and now we're just waiting on https://github.com/carrot/rooftop-node/pull/4

**Comment by hhsnopek** [comment]
requires `url` variable to be added to travis - cc @jescalan

**Comment by jescalan** [comment]
ok new `rooftop-node` version released!

**Comment by hhsnopek** [comment]
see #36

**Pull request closed without merging**